index.html

```diff
@@ -248,6 +248,7 @@ <h2 class="section-heading animated" data-animation="bounceInUp">Team</h2>
                      <div class="col-md-12">
                         <div class="row">
                            <div class="portfolio-items isotopeWrapper clearfix" id="4">
+
```

Comment:
There are extra spaces that have been added everywhere. While getting the diff, git also considers these spaces to be added as a change, so remove these spaces.